docs/index.md

```diff
@@ -1,43 +0,0 @@
-> [!CAUTION]
-> 
-> The `docs` folder has been moved to the centralized documentation repository, [docs-internal](https://github.com/scalar-labs/docs-internal). Please update this documentation in that repository instead.
-> 
-> To view the ScalarDL documentation, visit [ScalarDL Documentation](https://scalardl.scalar-labs.com/docs/).
```

Comment:
I'm not sure, but should we keep this to redirect users to the docs site? Or does the redirection work in other ways? Also, did you intentionally leave the `_includes` and `_layouts`?

Reply:
@jnmt 
> should we keep this to redirect users to the docs site? Or does the redirection work in other ways?

This isn't really an automatic redirect, so I don't think we need it. We have a link to the docs site in the **About** section of the [scalar-labs/scalardl](https://github.com/scalar-labs/scalardl) home page and in the [root README](https://github.com/scalar-labs/scalardl/blob/master/README.md), so I think that should be enough.

> Also, did you intentionally leave the _includes and _layouts?

Actually, we don't really need those anymore, so I just now removed them in https://github.com/scalar-labs/scalardl/pull/67/commits/d205338638017951e28b572dc318344ed722f99e.

Reply:
Got it, thank you for the update!